### ISSUE TITLE
Docker container will not start in Unraid

### ISSUE THREAD

**Issue by chogan72** [YoRyan/mailrise#118]
I'm running into issues getting the mailrise docker container to run in Unraid. I am running Unraid 6.12.10 with Docker 
24.0.9. When I try to start the container this is the error that I receive.

```
CRITICAL:mailrise.skeleton:No Apprise targets are configured
```

I know that Unraid is a bit different with mapping the config file, so I'm guessing it's something related to that. After looking at the solution to this issue #50, below is the current config that I have set up. 

**mailrise.conf**
```
configs:
  discord:
    urls:
      - discord://webhook
```

**config.yml**
```
version: '1.0'

services:
  mailrise:
    container_name: mailrise
    image: yoryan/mailrise:latest
    ports:
      - 8025:8025
    environment:
      - PUID=1000
      - PGID=1000
    volumes:
      - /mnt/user/appdata/mailrise/mailrise.conf:/etc/mailrise.conf

```

**docker run**
```
docker run
  -d
  --name='mailrise'
  --net='bridge'
  -e TZ="America/Chicago"
  -e HOST_OS="Unraid"
  -e HOST_HOSTNAME="Name"
  -e HOST_CONTAINERNAME="mailrise"
  -l net.unraid.docker.managed=dockerman
  -p '8025:8025/tcp'
  -v '/mnt/user/appdata/mailrise/':'/etc/mailrise':'ro' 'yoryan/mailrise:latest' /etc/mailrise/config.yml
```

I'm sure there is something simple that I am overlooking. Any ideas what I am doing wrong?

**Comment by simonmcnair** [comment]
you mailrise.conf looks wrong. discord://webhook needs to be replaced by your config.  It needs your specific discord://webhook_id/webhook_token

**Comment by chogan72** [comment]
My bad I should have added more detail there. That is how I have it formatted in the actual config file. I just put webhook there when I redacted the actual values. 

I don't remember exactly where I left this off, but I did try other services besides Discord as was getting the same error. I ended up deciding it was something I had configured strange on my Unraid server preventing the container from starting. I ended up writing a python script instead to route the messages.
